backend/btrixcloud/uploads.py

```diff
@@ -202,6 +202,9 @@ async def _create_upload(
                     org.id, file, crawl_id, "upload"
                 )
 
+        if collections:
+            await self.colls.update_crawl_collections(crawl_id)
```

Comment:
Should this occur after the `add_crawl_pages_to_db_from_wacz` call? I don’t know if this ends up as a race condition

Reply:
You are totally right! Modified to await `add_crawl_pages_to_db_from_wacz` first.